src/video.c

```diff
@@ -208,19 +208,21 @@ void init_framebuffer(s32 index) {
     }
     gVideoFbWidths[index] = gVideoModeResolutions[gVideoModeIndex & 7].width;
     gVideoFbHeights[index] = gVideoModeResolutions[gVideoModeIndex & 7].height;
+    // TODO: All of the gVideoFramebuffers values are suspicious and probably not the right code.
+    // The 0x3F is coincidentally sizeof(gVideoModeResolutions) - 1, but that might just be a coincidence
```

Comment:
It looks like it's just aligning `gVideoDepthBuffer` to a multiple of 0x40. I don't think it's related to gVideoModeResolutions.

Reply:
Seems like a reasonable explanation to me. I'll remove the comment.